opus/application/static_media/js/opus.js

```diff
@@ -794,10 +786,16 @@ var opus = {
             browserVersion = "0.0";
         }
 
-        let modalMsg = (`Your current browser is ${browserName} ${browserVersion}.
-                        Please update the browser. OPUS supports Firefox (66+),
-                        Chrome (74+), Safari (12.1+), Opera (58+), and Microsoft Edge
-                        (18+).`);
+        let browser = `${browserName} ${browserVersion}`;
+        if (browserName === "unsupported") {
+            browser = browserName;
+        }
+        let modalMsg = (`Your current browser is ${browser}.
+                        Please update the browser. OPUS supports
```

Comment:
One more change here. Right now it will say "Your current browser is unsupported. Please update the browser." That doesn't make much sense. Move the "Please update the browser." into the let statement at 789 and then it will work better.

Also let's change it to "Please update your browser."